<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraManager</class>
 <widget class="QWidget" name="CameraManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HTitleButton" name="camera2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="HTitleButton" name="camera1" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="HTitleButton" name="camera3" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HTitleButton</class>
   <extends>QWidget</extends>
   <header>HTitleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
